### ISSUE TITLE
Browsers Preview

### ISSUE THREAD

**Issue by vv-monsalve** [marcologous/hanken-grotesk#71]
Hi @marcologous, these are the Browsers preview images. The Thin weight looks too light.
Would it be possible to increase the weight value for that master? 

## Thin, ExtraLight, Light, Regular

**Windows 10 Chrome_71**
![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/48698976/133362903-07e984b7-3b3f-43c5-9a39-77620fa8c1a6.png)
**Windows 10 Edge_18**
![Desktop_Windows_10_edge_18 0_](https://user-images.githubusercontent.com/48698976/133362886-9bc9e337-6641-488c-94dc-e86fdde79059.png)
**OS X High Sierra Safari_11**
![Desktop_OS_X_High_Sierra_safari_11 1_](https://user-images.githubusercontent.com/48698976/133362896-1d012c6c-46f9-490f-b8d5-f9eb877bffe3.png)

**Comment by vv-monsalve** [comment]
## Medium, SemiBold, Bold, ExtraBold


**Windows 10 Chrome_71**
![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/48698976/133363130-befdbad2-1de1-4585-a3c4-5617179a517c.png)
**Windows 10 Edge_18**
![Desktop_Windows_10_edge_18 0_](https://user-images.githubusercontent.com/48698976/133363117-926dd2e8-edea-4c11-8722-68e6ff5c268d.png)
**OS X High Sierra Safari_11**
![Desktop_OS_X_High_Sierra_safari_11 1_](https://user-images.githubusercontent.com/48698976/133363126-9078bd05-ad8c-4ac6-b961-a010a6ce71ec.png)

**Comment by marcologous** [comment]
Yes! I can change the master. It is way too thin indeed @vv-monsalve

**Comment by vv-monsalve** [comment]
> Yes! I can change the master. It is way too thin indeed @vv-monsalve

Hi @marcologous, how are you coming along with Hanken? Do you have any date in mind to have the new source files with the improvement for the Thin weight?

**Comment by marcologous** [comment]
Hi Viviana!


Apologies, I was on vacation until November 4th and just got back to my
workflow now. I am currently working on weight redistribution and making
the thin weight thicker. It will be ready tomorrow. Also, taking the black
weight fixes into consideration :)


Thanks,

Alfredo Marco Pradil
+971561153745

**Comment by marcologous** [comment]
Hi @vv-monsalve ,

Merged the updates with the Master :) Let me know how it goes!

**Comment by vv-monsalve** [comment]
Hi @marcologous,

Apologies for the belated response. It has been quite an agitated end and beginning of the years.

I've pulled the files at 258f44cf0ee4772064278741fa0f58dd426e0f1f, made some small fixes and produced new fonts.
These are the resulting images.

Thin weight looks better now.

## Thin, ExtraLight, Light, Regular

**Windows 10 Chrome_71**
![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/48698976/155806191-09e59d36-2d11-484a-9e7e-9fd2730fcb7c.png)
**Windows 10 Edge_18**
![Desktop_Windows_10_edge_18 0_](https://user-images.githubusercontent.com/48698976/155806195-36c00d0d-9066-4980-9dff-18b2827c6e8a.png)
**OS X High Sierra Safari_11**
![Desktop_OS_X_High_Sierra_safari_11 1_](https://user-images.githubusercontent.com/48698976/155806185-b9ee55b1-ea5c-4d5c-9e81-b86d073949be.png)

**Comment by vv-monsalve** [comment]
Hi @marcologous, I'm closing this issue since it was completed a long time ago :)
